frontend/src/store/modules/website/mutations.js

```diff
@@ -20,11 +21,12 @@ export default {
     [SET_CURRENT_WEBSITE]: (state) => {
         state.isCurrentWebsite = true;
     },
-    [UPDATE_CURRENT_WEBSITE]: (state, website) => {
-        state.currentWebsite = {
-            ...state.currentWebsite,
-            ...website
-        };
+    [UPDATE_CURRENT_WEBSITE]: (state, name) => {
+        state.relateUserWebsites.map(website => {
+            if(website.id === state.selectedWebsite) {
+                website.name = name;
```

Comment:
Don't update object like this, it can break reactivity. Instead - find needed object, and out new array instance in store